### DISCUSSION TITLE
How to give a different color to keywords as \item and similars?

### DISCUSSION THREAD

**Discussion by rafa4mat** [WinEdt-Team/WinEdt-Support#89] [1.1 General Help and Discussion]
1,5 years ago I try this but I can´t reach the solution. I can´t remeber if it´s possible or not.

Is possible give a different color to words like \item ?,   (Alex help me to reach the solution with the & symbols in a table, using Option Interface / Highlighting Effects / Filter Sets).

![snap004379](https://github.com/user-attachments/assets/f30b97f3-66c9-4599-9004-be99f17c6158)

I think that it would be possible in Option Interface / Highlighting Effects / Filter Sets OR in Option Interface / Highlighting Effects / Keywords.

Excuse me if I asked this close to 2 years ago.

I tried adding this at the end Highlighting Effects: Colors,... / Keywords  this code but no success

    KEYWORD_GROUP="LaTeX Enviroment Subparts"  //  GROUP Added 
    ENABLED=1
    MODE_FILTER="TeX;BibTeX;MetaPost"
    BEFORE="\ (single)"
    AFTER="~Alpha@"
    BOLN_NOT_OK=1
    EOLN_NOT_OK=0
    HIGHLIGHT_START=1
    STRICT_PRIORITY=0
    PRIORITY=9   // Because item is in the list of KEYWORD_GROUP="WinEdt MUI Properties" with priority 8
    CASE_SENSITIVE=1
       TEXT_COLOR="kwcLaTeXEnvSubparts"    // adding kwcLaTeXEnvSubparts="#CB4B16" in the theme I use.
       DEFAULT_FONT=1
    LIST="END_LIST"
    item
    END_LIST

**Comment by saxyx** [comment]
The list contained in `KEYWORD_GROUP="[La]TeX Keywords"` also includes `item`. Placing the code before `KEYWORD_GROUP="[La]TeX Keywords"` will make the code take effect.

**Comment by rafa4mat** [comment]
Thank you very much, saxyx.

I didn´t realise that 'item' in [La]TeX Keywords. 

It works fine as you explain or commenting previous 'item'.

![snap004382](https://github.com/user-attachments/assets/16c5d5fb-2178-42a7-9540-c2f794f39c1f)
